### DISCUSSION TITLE
Agenda 2022-01-18

### DISCUSSION THREAD

**Discussion by afoxman** [microsoft/rnx-kit#996] [Bundle Working Group]
Topics:
* Metro as a web bundler
* Using the metro cache with S3 (*afoxman*) - [notes](https://github.com/microsoft/rnx-kit/discussions/983)
* Haste-map replacement experiments - [notes](https://github.com/microsoft/rnx-kit/discussions/992)
* Bundle-size reduction using tree-shaking and bundle-splitting (*tido64*)

**Comment by kelset** [comment]
two potential other points to touch on:
* https://github.com/microsoft/rnx-kit/discussions/1515
* https://twitter.com/Baconbrix/status/1525853511551832064
